spinta/datasets/commands/check.py

```diff
@@ -1,10 +1,31 @@
+import pathlib
+
 from spinta import commands
-from spinta.components import Context
+from spinta.components import Context, Config, Store
 from spinta.datasets.components import Dataset, Resource
+from spinta.exceptions import MultipleDatasetsError
+from spinta.exceptions import InvalidFileName
 
 
 @commands.check.register()
 def check(context: Context, dataset: Dataset):
+    config: Config = context.get('config')
+    store: Store = context.get('store')
+
+    filename = config.check_filename
+    if filename:
+        if len(store.manifest.datasets) > 1:
+            raise MultipleDatasetsError(manifest=store.manifest)
```

Comment:
It looks, that this check must be done on `load` command, because multiple dataset files passed via command line arguments are combined into a single in-memory manifest. So there are two options:

1. Refactor code base, to work on multiple manifests, instead of a single one. This might be difficult to do, but a more correct solution.
2. Do the checking on `load` command, when files are read one by one. This is a less correct, but easier to do.

So probably, for this task second option is a better choice.